.gitignore

```diff
@@ -157,3 +157,6 @@ sftp-config.json
 
 .env
 dist/
+
+## graphql - generated code
```

Comment:
any more context on what this is? I'm not seeing these locally - is it something your editor added?